docs/api.md

```diff
@@ -1161,7 +1161,7 @@ Dialog's type, could be one of the `alert`, `beforeunload`, `confirm` and `promp
 [ConsoleMessage] objects are dispatched by page via the ['console'](#event-console) event.
 
 #### consoleMessage.args
-- <[Array]<[string]>>
+- <[Array]<[JSHandle]>>
```

Comment:
FWIW, this link is somehow broken here and in some other places.

Reply:
@vsemozhetbyt thanks, fixed in https://github.com/GoogleChrome/puppeteer/commit/3214bb73b05ea281bf49341e4e81cb8dc489a864